cmd/thanos/query_frontend.go

```diff
@@ -285,6 +290,12 @@ func runQueryFrontend(
 		return errors.Wrap(err, "error validating the config")
 	}
 
+	if cfg.EnableXFunctions {
```

Comment:
can we just loop over parse.XFunctions instead, that way we dont need to know the actual function names at all.

Reply:
Done.